internal/pkg/docker/dockerengine/dockerengine.go

```diff
@@ -138,14 +133,23 @@ func (c CmdClient) Build(in *BuildArguments) error {
 	}
 
 	args = append(args, dfDir, "-f", in.Dockerfile)
-	// If host platform is not linux/amd64, show the user how the container image is being built; if the build fails (if their docker server doesn't have multi-platform-- and therefore `--platform` capability, for instance) they may see why.
-	if in.Platform != "" {
-		log.Infof("Building your container image: docker %s\n", strings.Join(args, " "))
+	return args, nil
+}
+
+type dockerConfig struct {
+	CredsStore  string            `json:"credsStore,omitempty"`
+	CredHelpers map[string]string `json:"credHelpers,omitempty"`
+}
+
+// Build will run a `docker build` command for the given ecr repo URI and build arguments.
+func (c CmdClient) Build(in *BuildArguments) error {
```

Comment:
If `CmdClient` is specifically designed for running `docker` commands, can we rename it to `DockerCmdClient`?